pkg/ttl/cache/task.go

```diff
@@ -116,6 +116,9 @@ type TTLTaskState struct {
 	ErrorRows   uint64 `json:"error_rows"`
 
 	ScanTaskErr string `json:"scan_task_err"`
+
+	// When PreviousOwner != "", it means this task is resigned from another owner
+	PreviousOwner string `json:"prev_owner,omitempty"`
```

Comment:
How about also print a log in the `lockScanTask` if the `owner` or `previousOwner` is not `NULL`?

It seems that this field is not used by the code logic, so it's only useful for the user to observe the situation, right?